COMPILE

```diff
@@ -1,57 +1,2 @@
-Compiling Mudlet on Linux
--------------------------
-
-Use the Qt-4.6-SDK package from ftp://ftp.trolltech.com/qtsdk/ or http://qt.nokia.com
-It comes with a binary distribution of the libs a ready made compiler for windows (mingw)
-(NOTE GNU/Linux: If you experience instability or weird behaviour of Mudlet link against the SDK libs instead of your distro's qt libs.)
-
-1. DEPENDENCIES:
-================
-required libs to compile Mudlet:
-- Qt >=4.4 (included in the SDK)
-- liblua-5.1
-- libpcre-7.8 or later
-- yajl-1.0 (you can also use yajl-2.0 if you change make config file src/src.pro and include lua_yajl.c instead of lua_yajl1.c)
-- hunspell
-- boost (www.boost.org)
-- zlib.1.2.3
-- Open GL libs + Qt OpenGL module see Qt OpenGL docs for more info
-- zzlib (http://zziplib.sourceforge.net)
-- luazip (Lua unzip module) http://www.keplerproject.org/luazip/
-- Lua File System (LFS)
-- rex_pcre (Lua pcre interface)
-- Lua sql (Lua module for SQL)
-
-(NOTE: GNU/Linux: You'll always need the "dev" versions of the required libs if you use packages of your distro.)
-
-additional packages that should be present on your system to get the full feature set of Mudlet:
-(NOTE: Mudlet checks if these pacakges are available at runtime and can do without them if they cannot be found in the Lua path of your system. Check Mudlet's startup messages when connecting to see if Mudlet can find these extra packages.))
-- luasql (http://www.keplerproject.org/luasql/)
-- lrexlib (http://lrexlib.luaforge.net/) NOTE: You only need to compile the pcre module. The other modules of this lib are not used.
-- sqlite3
-
-
-2. COMPILING:
-=============
-  Use qmake. You can adjust the paths to the dependencies in the project file
-   mudlet/src/src.pro - simply look at the top of the file and adjust the paths you find to
-   your own system. This is easy. If compile breaks with an error your paths are not correct.
-   After having adjusted src.pro:
-   make clean
-   qmake-qt4 (NOTE: On most systems there is qmake and qmake-qt4 qmake relates to the old qt version 3 qmake.)
-   make
-   make install
-
-  Note: If you run into compiler problems then edit the file mudlet/src/src.pro in order to
-   change the INCLUDE and/or LIBS part of the make file and set the correct path on your system.
-
-Compiling Mudlet on Macs
-------------------------
-These instructions are rough and in-progress.
-
-Install Homebrew (it'll require XCode - install that), With Homebrew, install:
-  boost, hunspell, libzzip, luarocks, pcre, readline, cmake, libzip, lua, lzlib, pkg-config, yajl
-
-Then run qmake, and the usual make to compile.
-
-See INSTALL instructions on how to get the finished .dmg from a .app that you've compiled.
+Please see http://wiki.mudlet.org/w/Compiling_Mudlet for latest Mudlet compile
+instructions on Windows, macOS, and Linux distributions.
```

Comment:
They're called macOS now: [Apple Renamed Its Desktop Operating System OS X to macOS](http://fortune.com/2016/06/13/apple-mac-os-sierra/).

I don't know if there is a standard around this. I was thinking we could export the wiki page into source every month or so to keep it more or less relevant, but that's more work than I've got time for so this is a quick & small improvement.